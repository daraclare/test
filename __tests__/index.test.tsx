import React from "react";
import { render, fireEvent } from "@testing-library/react";
import TestRenderer from "react-test-renderer";
import Index from "../pages/index";

describe("Homepage", () => {
  it("should render without throwing an error", () => {
    const { container } = render(<Index />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
