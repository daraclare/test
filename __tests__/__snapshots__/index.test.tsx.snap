// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage should render without throwing an error 1`] = `
<div>
  <nav
    class="sc-bdVaJa khUSez"
  >
    <ul
      class="sc-bwzfXH hxBrhe"
    >
      <li
        class="sc-htpNat hDYkct"
      >
        <a
          class="sc-bxivhb hscBq"
        >
          Home
        </a>
      </li>
      <li
        class="sc-htpNat hDYkct"
      >
        <a
          class="sc-bxivhb hscBq"
        >
          About
        </a>
      </li>
      <li
        class="sc-htpNat hDYkct"
      >
        <a
          class="sc-bxivhb hscBq"
        >
          Contact
        </a>
      </li>
    </ul>
  </nav>
  <main
    class="sc-ifAKCX fAUuoW"
  >
    Your user agent: 
  </main>
</div>
`;
